src/components/atoms/ProfileFront.jsx

```diff
@@ -69,7 +69,7 @@ function Front (props) {
             variant="body2"
             style={{ fontWeight: "300", lineHeight: "1.2" }}
           >
-            {title}
+            <p dangerouslySetInnerHTML={{__html : title}} />
```

Comment:
Programming in the danger zone. 
![image](https://user-images.githubusercontent.com/1295846/101684429-3dc85800-3a1b-11eb-9dc9-4565981ba095.png)